lib/ap_configurator/scripts/activate_brcm_minimal.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
```

Comment:
Same here, use Use !/usr/bin/env bash

(sh might actually cause also problems on iOS and some linux distros) - potentially let's move most scripts into main bin folder and maybe give them stand-alone functionality and help